### PR TITLE
Addnet45

### PR DESCRIPTION
**Pull request by adaggarwal**
Fixes part of [#1298](https://github.com/dotnet/source-build/issues/1298)

### CODE REVIEW

File: src/targetPacks/ILsrc/microsoft.netframework.referenceassemblies.net461/1.0.0-alpha-5/microsoft.netframework.referenceassemblies.net461.csproj [hunk 1]
```diff
@@ -1,8 +1,8 @@
 <Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
     <TargetFramework>netstandard2.0</TargetFramework>
-    <OutputPath>$(ArtifactsBinDir)microsoft.netframework.referenceassemblies.net461/1.0.0/</OutputPath>
-    <BaseIntermediateOutputPath>$(ArtifactsObjDir)microsoft.netframework.referenceassemblies.net461/1.0.0/</BaseIntermediateOutputPath>
+    <OutputPath>$(ArtifactsBinDir)microsoft.netframework.referenceassemblies.net461/1.0.0-alpha-5/</OutputPath>
```
Comment (dseefeld) [inline src/targetPacks/ILsrc/microsoft.netframework.referenceassemblies.net461/1.0.0-alpha-5/microsoft.netframework.referenceassemblies.net461.csproj]: Same question here.  Why is this version changing?

Comment (adaggarwal) [reply src/targetPacks/ILsrc/microsoft.netframework.referenceassemblies.net461/1.0.0-alpha-5/microsoft.netframework.referenceassemblies.net461.csproj]: The version in previous updates was wrong. I realized it this time that its actually `alpha-5` which source-build-reference-packages references.

File: src/targetPacks/ILsrc/microsoft.netframework.referenceassemblies.net46/1.0.0-alpha-5/microsoft.netframework.referenceassemblies.net46.csproj [hunk 2]
```diff
@@ -1,8 +1,8 @@
 <Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
     <TargetFramework>netstandard2.0</TargetFramework>
-    <OutputPath>$(ArtifactsBinDir)microsoft.netframework.referenceassemblies.net46/1.0.0/</OutputPath>
-    <BaseIntermediateOutputPath>$(ArtifactsObjDir)microsoft.netframework.referenceassemblies.net46/1.0.0/</BaseIntermediateOutputPath>
+    <OutputPath>$(ArtifactsBinDir)microsoft.netframework.referenceassemblies.net46/1.0.0-alpha-5/</OutputPath>
```
[the remark this answers is not in the document]
Comment (adaggarwal) [reply src/targetPacks/ILsrc/microsoft.netframework.referenceassemblies.net46/1.0.0-alpha-5/microsoft.netframework.referenceassemblies.net46.csproj]: Same reason as `net461`

File: src/targetPacks/ILsrc/microsoft.netframework.referenceassemblies.net461/1.0.0-alpha-5/microsoft.netframework.referenceassemblies.net461.csproj (hunk 1 above)
Comment (dseefeld) [reply src/targetPacks/ILsrc/microsoft.netframework.referenceassemblies.net461/1.0.0-alpha-5/microsoft.netframework.referenceassemblies.net461.csproj]: Got it.  Thanks!

**Pull request merged**